docs/mnist_example/mnist.cpp

```diff
@@ -299,6 +302,9 @@ int main() {
     }
   }
 
+  // Free the output as it is no longer needed.
+  omTensorListDestroy(tensorListIn);
```

Comment:
copy-and-paste error. Should be `tensorListOut`.

Reply:
Yes on both counts. Thanks for catching!